data/categories.txt

```diff
@@ -1191,7 +1191,7 @@ tourism-information
 en:4tourist information|4information|sights|U+2139|U+1F481
 ru:4туринформация|информация|достопримечательность
 uk:4турінформація|інформація|пам’ятка|пам’ятні місця
-de:4Information|Sehenswürdigkeit
+de:4Information|Tourist-Information
```

Comment:
You can swap these words, so users will se first one on a screen (and second and others will be there as search synonyms).